src/tracing/metric_keys.h

```diff
@@ -29,16 +29,7 @@ typedef const char * MetricKey;
 /**
  * List of supported metric keys
  */
-constexpr MetricKey kMetricDiscoveryOverBLE      = "disc-over-ble";
-constexpr MetricKey kMetricDiscoveryOnNetwork    = "disc-on-nw";
-constexpr MetricKey kMetricPASESession           = "pase-session";
-constexpr MetricKey kMetricPASESessionPair       = "pase-session-pair";
-constexpr MetricKey kMetricPASESessionBLE        = "pase-session-ble";
```

Comment:
These keys should have been removed from the initial PR that put the metric framework up. These are not used and we should only add keys that are actually used commonly in the SDK here.